components/camel-weather/src/main/java/org/apache/camel/component/weather/WeatherConfiguration.java

```diff
@@ -342,4 +353,61 @@ public String getProxyAuthHost() {
     public void setProxyAuthHost(String proxyAuthHost) {
         this.proxyAuthHost = proxyAuthHost;
     }
+
+    public String getZip() {
+        return zip;
+    }
+
+    /**
+     * Zip-code, e.g. 94040,us
+     */
+    public void setZip(String zip) {
+        this.zip = zip;
+    }
+
+    public List<String> getIds() {
+        return (List<String>) ids;
+    }
+
+    public void addId(String id) {
+        if (ids == null) {
+            ids = new ArrayList<>();
+        }
+        ids.add(id);
+    }
+    /**
+     * List of id's of city/stations
+     */
+    public void setIds(String... ids) {
```

Comment:
Make a setter that takes a single String where the values are separated by comma so you can pass in the value in the same string. This makes it easier if loading values from properties/config/tooling